tensorrt_llm/_torch/visual_gen/models/glm_image/transformer_glm_image.py

```
@@ -0,0 +1,859 @@
+            ]
+        )
+
+    def forward(
+        self,
+        hidden_states: torch.Tensor,
+        encoder_hidden_states: Optional[torch.Tensor] = None,
+        attention_mask: Optional[torch.Tensor] = None,
+        image_rotary_emb: Optional[Tuple[torch.Tensor, torch.Tensor]] = None,
+    ):
+        batch_size, text_seq_length, embed_dim = encoder_hidden_states.shape
+        batch_size, image_seq_length, embed_dim = hidden_states.shape
```

Ruff (0.15.20)

[warning] Unpacked variable `embed_dim` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)